blocks/linkable-cards/linkable-cards.css

```diff
@@ -12,15 +12,40 @@
   padding-bottom: 60px;
 }
 
+/* this is considered an eyebrow style */
+.linkable-cards-container h4 {
```

Comment:
Why not `.linkable-cards h4`?

Reply:
There is more than 1 location of H4.
![image](https://user-images.githubusercontent.com/3883795/228011923-a533dd75-8182-42cf-b348-311affe280e9.png)